### ISSUE TITLE
Put enemy speed at 0 cause Unity crash or errors.

### ISSUE THREAD

**Issue by majausone** [suriyun-production/mmorpg-kit-docs#224]
![YhU2MfSjYB](https://user-images.githubusercontent.com/43614306/74362128-0cadd580-4dc0-11ea-8d95-91f1f8b90846.png)
Tested on Unity 2019.3 and 2019.2.

**Comment by insthync** [comment]
Thank you for bug report,
You can fix it by change some codes in BaseCharacterEntity.cs, find for `MoveAnimationSpeedMultiplier` property (Line 100) then change it to
```
public override sealed float MoveAnimationSpeedMultiplier { get { return this.GetCaches().BaseMoveSpeed > 0f ? GetMoveSpeed(MovementState, ExtraMovementState.None) / this.GetCaches().BaseMoveSpeed : 1f; } }
```

**Comment by JestHWas** [comment]
This fixes it for me. Will this be in the new update?

**Comment by insthync** [comment]
Fixed in 1.53b, if the problem still occurring, you can reopen it.
